README.md

```diff
@@ -118,118 +116,104 @@ The plugin will write out all metrics serialized as JSON to the specified file.
 See full task manifests using the mock and psutil plugins which are available in [examples/tasks](examples/tasks)  
 
 
-Example of running mock collector plugin, passthru processor plugin, and writing data as a JSON to file:
+Example of running [psutil collector plugin](https://github.com/intelsdi-x/snap-plugin-collector-psutil), [movingaverage processor plugin](https://github.com/intelsdi-x/snap-plugin-processor-movingaverage), and writing data as a JSON to file:
 
-Make sure that your `$SNAP_PATH` is set, if not:
+Set up the [Snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
+
+Make sure that [snapd](https://github.com/intelsdi-x/snap/blob/master/docs/SNAPD.md) and [snapctl](https://github.com/intelsdi-x/snap/blob/master/docs/SNAPCTL.md) are installed.
+
+In one terminal window, run the Snap daemon (in this case with logging set to 1 and trust disabled):
 ```
-$ export SNAP_PATH=<snapDirectoryPath>/build
+$ snapd -l 1 -t 0
 ```
-In one terminal window, open the snap daemon (in this case with logging set to 1 and trust disabled):
+
+In another terminal window:
+
+Get snap-plugin-publisher-file plugin:
 ```
-$ $SNAP_PATH/bin/snapd -l 1 -t 0
+wget http://snap.ci.snap-telemetry.io/plugins/snap-plugin-publisher-file/latest/linux/x86_64/snap-plugin-publisher-file
 ```
-In another terminal window:  
+
 Load snap-plugin-publisher-file plugin:
 ```
-$ $SNAP_PATH/bin/snapctl plugin load build/rootfs/snap-plugin-publisher-file
-
-Plugin loaded
-Name: file
-Version: 2
-Type: publisher
-Signed: false
-Loaded Time: Mon, 25 Jul 2016 12:30:25 CEST
+$ snapctl plugin load snap-plugin-publisher-file
```

Comment:
There is known problem regarding name collision with `snapd` package on ubuntu >= 16.xx, currently mentioned in Snap's readme. ~~Maybe this line should have absolute path?~~ Just checked, this is more complicated, I suggest holding on this.

All subsequent lines referring to snap binaries have same problem, but I don't want to spam with `Same as above.` comments :face_with_head_bandage: